db_stress_tool/multi_ops_txns_stress.cc

```diff
@@ -719,7 +717,6 @@ Status MultiOpsTxnsStressTest::SecondaryKeyUpdateTxn(ThreadState* thread,
     }
     auto& key_gen = key_gen_for_c_[thread->tid];
     key_gen->UndoAllocation(new_c);
-    RollbackTxn(txn).PermitUncheckedError();
```

Comment:
I think the stress test flag `FLAGS_rollback_one_in` loses its meaning now as we do not do rollback here anymore.

Reply:
Nice catch. Thanks.

It was pretty meaningless before this change, IMHO, because rollback was only tested immediately before destruction. Rollback is pointless unless you're reusing the iterator, which was untested. I'll do this in a follow-up PR.